src/assets/scss/foundations.scss

```diff
@@ -284,11 +285,6 @@ ol {
     }
 }
 
-p:empty {
```

Comment:
Is this removed intentionally?

Reply:
Yes sir. This is already there https://github.com/eslint/eslint.org/blob/337f2bc851f9361c7802d87f9b117aee974e11c7/src/assets/scss/foundations.scss#L169-L172